console/src/components/Services/Data/utils.js

```diff
@@ -375,6 +377,7 @@ FROM
     type: 'run_sql',
     args: {
       sql: runSql,
+      read_only: checkFeatureSupport(READ_ONLY_RUN_SQL_QUERIES) ? true : false,
```

Comment:
if instead of `false` we set it to `undefined` will this key be dropped at the client side itself?

Reply:
No real need to drop it as the readOnly flag is already supported. it's just the select along with with run_sql part that needs to be behind version check